x-pack/plugins/transform/public/app/common/validators/is_json_string.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Validates whether string input can be parsed as a valid JSON
+ * @param value User input value.
+ */
+export function isJsonString(value: unknown): boolean {
+  if (typeof value !== 'string') return false;
+
+  try {
+    return !!JSON.parse(value);
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(`JSON is invalid.\n${e}`);
```

Comment:
do we need to log it?

Reply:
removed it in [eb68c06](https://github.com/elastic/kibana/pull/173736/commits/eb68c06de6aba3d2ab537b1a9538b7faf8db33e6).